### DISCUSSION TITLE
Status of the integration

### DISCUSSION THREAD

**Discussion by liquidand** [Andre0512/hon#214] [General]
Is the development of this integration still active?

**Comment by deezid** [comment]
Wondering as well, just got a new washer from Haier which I'm otherwise satisfied with.

**Reply by MikeGuest** [reply]
No visible activity from the developer since the end of march. 

did you manage to get the integration to install? I just got that the application doesn't support gui setup and a fail. 

I also am trying to get my new washer to function - it's starting to look like I'll have to go back to the old style monitoring of a zigbee socket for power utilisation which i was trying to avoid. That was the whole point of buying a hOn compatible washer in the first place.

**Reply by liquidand** [reply]
I managed to have the integration installed an working. Since the project is discontinued I hope no problem will raise in the future

**Reply by deezid** [reply]
Also got it working after many full reboots of Home Assistant and the washer. 0.13 seems to work fine after that.

**Reply by MikeGuest** [reply]
Thanks. Am I missing something? (Like do  i have to install the python stuff separately?) as I just get the message that the integration can't be set up in the gui. My HA is up to date.

Thanks

**Reply by raymondjstone** [reply]
I don't recall having to install anything else, certainly not python

**Reply by deezid** [reply]
No other integration necessary. 
But don't install version 0.14 beta. It won't really connect.

**Comment by raymondjstone** [comment]
last beta was 2nd week of April, so has been some activity after March, I would doubt that the work has halted given the initial backlash againsy Haier when they tried to kill it, but maybe Andre is ill, busy working, or has some personal reason for not having any recent time. If it remains inactive I would suspect someone will take a fork of it and start developing that
